### DISCUSSION TITLE
Move "Other Formats"

### DISCUSSION THREAD

**Discussion by justanothergithubber** [quarto-dev/quarto-cli#3798] [Feature Requests]
Is it possible to move "Other Formats" that is normally located in the margin sidebar at the right to the left? (like Table of Contents which is controlled by `toc-location`) This gives more space for main content when `page-layout` is set to `full`.

**Comment by dragonstyle** [comment]
Thanks for the suggestion - I will take a look at this and see what we can come up with!

**Comment by mcanouil** [comment]
Closing in favour of https://github.com/quarto-dev/quarto-cli/issues/3895
